tailwind.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
it would be nice to have jit mode enabled as it would result in a lot smoother dev experience. Without the jit mode, it would produce a huge CSS file even if we're using just two or three classes and it could pump up the fans of most machines.